person.rb

```diff
@@ -2,14 +2,19 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- rentals can also be modified in this class `attr_accessor :name, :age, :rentals`